apps/src/templates/rubrics/RubricContainer.jsx

```diff
@@ -70,6 +71,59 @@ export default function RubricContainer({
       });
   };
 
+  useEffect(() => {
+    if (!!studentLevelInfo) {
+      const studentId = studentLevelInfo.user_id;
+      const rubricId = rubric.id;
+      const dataUrl = `/rubrics/${rubricId}/get_ai_evaluations?student_id=${studentId}`;
+
+      fetch(dataUrl)
+        .then(response => {
+          if (!response.ok) {
+            throw new Error('Network response was not ok');
+          }
+          return response.json();
+        })
+        .then(data => {
+          setAiEvaluations(data);
+        })
+        .catch(error => {
+          console.log(
+            'There was a problem with the fetch operation:',
+            error.message
+          );
+        });
+    }
+  }, [rubric.id, studentLevelInfo]);
+
+  const getAiUnderstanding = learningGoalId => {
+    if (!!aiEvaluation) {
+      const aiInfo = aiEvaluation.find(
+        item => item.learning_goal_id === learningGoalId
+      );
+      if (!!aiInfo) {
+        return aiInfo.understanding ? aiInfo.understanding : null;
```

Comment:
nit: I think this can be simplified to `return aiInfo.understanding`

Reply:
Actually, you may be able to simplify a lot of this to be
```
return aiInfo?.understanding
```

(same below for confidence)